projects/test-app/src/hslayers-app/hslayers-app.component.ts

```diff
@@ -436,7 +436,7 @@ export class HslayersAppComponent {
         {name: 'warning', url: '/assets/icons/warning.svg'},
         {name: 'wifi', url: '/assets/icons/wifi8.svg'},
       ],
-      status_manager_url: 'http://localhost:8086',
+      share_service_url: 'http://localhost:8086',
```

Comment:
```suggestion
      shareServiceUrl: 'http://localhost:8086',
```

Let's go camelCase when we are changing it already (applies also to other occurences). What do you think?

Reply:
Right, thats a great idea!